std-bits/base/src/main/java/org/enso/base/ObjectComparator.java

```diff
@@ -1,73 +1,69 @@
 package org.enso.base;
 
+import org.graalvm.polyglot.Context;
 import org.graalvm.polyglot.Value;
 
 import java.time.LocalDate;
 import java.time.LocalTime;
 import java.time.ZonedDateTime;
 import java.util.Comparator;
 import java.util.Locale;
-import java.util.function.BiFunction;
 import java.util.function.Function;
+import java.util.function.BiFunction;
 
 public class ObjectComparator implements Comparator<Object> {
-  private static ObjectComparator INSTANCE;
-
-  /**
-   * A singleton instance of an ObjectComparator.
-   *
-   * @param fallbackComparator this MUST be the default .compare_to function for Enso. Needs to be
-   *     passed to allow calling back from Java.
-   * @return Comparator object.
-   */
-  public static ObjectComparator getInstance(Function<Object, Function<Object, Value>> fallbackComparator) {
-    if (INSTANCE == null) {
-      INSTANCE = new ObjectComparator(fallbackComparator);
+  public static final ObjectComparator DEFAULT = new ObjectComparator();
+
+  private static BiFunction<Object, Object, Integer> EnsoCompareCallback = null;
+
+  private static void initCallbacks() {
+    if (EnsoCompareCallback == null) {
+      var module = Context.getCurrent().getBindings("enso").invokeMember("get_module", "Standard.Base.Data.Ordering");
+      var type = module.invokeMember("get_type", "Comparable");
```

Comment:
This kind of poking around in Enso doesn't make me particularly happy.

However the only alternative I can offer is to always make sure each call from Enso to Java (that needs this callback) passes the `compare_callback` function to Java.

Reply:
I also don't like such a poking in Enso. But I guess it is OK for this PR, as a temporary solution.